ledger/ledger-api-auth/src/test/suite/scala/com/digitalasset/ledger/api/auth/AuthServiceJWTCodecSpec.scala

```diff
@@ -48,6 +58,71 @@ class AuthServiceJWTCodecSpec
         minSuccessful(100))(
         value => serializeAndParse(value) shouldBe Success(value)
       )
+
+      "support legacy sandbox format" in {
```

Comment:
The new format with the OIDC namespace is handled by the `work for arbitrary values` test case?

Reply:
Correct, but I have now also added an explicit test for the new format.